README.md

```diff
@@ -14,22 +14,43 @@ Dazzle Cerashop is an E-commerce website which sells ceramic products.
 
 ## Introduction
 
-- Dazzle Cerashop was built with an idea to sell products that has less harmful impact on environment.
+- Dazzle Cerashop is built with an idea to sell products that has less harmful impact on environment.
```

Comment:
Just a nitpick, but the grammatically correct sentence would be `...products that have less harmful....`

Reply:
Ohh..yes Thanks! Fixed it